MapboxNavigation/NavigationMapView.swift

```diff
@@ -719,6 +722,55 @@ open class NavigationMapView: MGLMapView {
         }
     }
     
+    public func showVoiceInstructionsOnMap(routeProgress: RouteProgress) {
+        guard let style = style else {
+            return
+        }
+        
+        let instructions = Array(routeProgress.route.legs.flatMap {
```

Comment:
Why not pass in the Route directly? Is anything else on RouteProgress relevant to this feature?